assets/static/js/mapWizard.js

```diff
@@ -26,13 +26,7 @@ function MapWizard(json)
     var diaHeight = $(window).height()*.8;
 
     $.ajaxSetup({
-        timeout: 60000,
-        error: function(x, t, m) {
-            var message;
-            if (typeof m.message != "undefined")
-            {message = m.message;}else{message = m;};
-            console.log("Error in Ajax:"+t+" Message:"+message);
-        }
+        timeout: 20000,
```

Comment:
Maybe should be just 10 seconds, or less.

Reply:
I think 10 sec sounds reasonable.